app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsUserStats.kt

```diff
@@ -115,7 +115,7 @@ object SuggestedEditsUserStats {
                                 }
                             }
                         }
-                        totalPageViews
+                        resultList
```

Comment:
Since the logic above will be used somewhere, it would be better if we can create two public methods:

`getPageViewsObservable` which keeps our existing logic.
`getPageViewsListObservable` which can be use in the contribution screen.